alembic/versions/20250607181932_54080d3503e2_make_role_id_bigint.py

```diff
@@ -0,0 +1,43 @@
+"""Make role_id bigint
```

Comment:
this got pulled in from https://github.com/tribesonecommunity/discord-bots/pull/142